Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -1,72 +1,55 @@
 ﻿using System;
-using System.IO;
 
 namespace BootCamp.Chapter
 {
-    public class Credentials
+    public readonly struct Credentials : IEquatable<Credentials>
```

Comment:
Love the use of interface and readonly struct. Even though it wasn't really a requirement, it certainly fits there.